microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -58,6 +58,10 @@ def ltp_lite(
         tests = variables.get("ltp_test", "")
         skip_tests = variables.get("ltp_skip_test", "")
 
+        # block device is required for few ltp tests
+        # If not provided, we will find a disk with enough space
+        block_device = variables.get("block_device", None)
```

Comment:
`ltp_drive_name`

Reply:
`lip_drive_name` seems related to installation drive. How about adding a test prefix like `test_block_device`?

Reply:
1. It needs start with "ltp_" to get others know it applies to ltp test cases.
2. Disk is not the only block devices in Linux. I see you assigned it to an internal variable name "drive_name", so I set it to second part to make it consistent.
3. I just found the `device_name` is set to the file system type "-z" actually. Does it really work? It looks should be set to tmp folder "-d".

How about name it like "ltp_tmp_dir"?

Reply:
There are two different things here:

```
# add logging and output file parameter
if drive_name:
    parameters += f"-z {drive_name} "

 # directory where temporary files will be created
parameters += f"-d {temp_dir} "
```

1. `drive_name`: This is set with `-z` option. This is used by some tests, probably to measure performance. 
2. `temp_dir`: This is set with `-d` option. This is used for storing temp files during ltp run. 

I will update the code to make it `ltp_block_device` and set `block_device` to `-z` option to be consistent.

Reply:
From LTP manpage:

```
-z BIG_DEVICE   Some tests require a big unmounted block device
                    to run correctly.
 -d TMPDIR       Directory where temporary files will be created.
```